examples/machine-vsphere.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  # If you change the namespace/name, you must also
+  # adjust the rbac rules
+  name: machine-controller-vsphere
+  namespace: kube-system
+type: Opaque
+stringData:
+  password: << VSPHERE_PASSWORD >>
+---
+apiVersion: "machine.k8s.io/v1alpha1"
+kind: Machine
+metadata:
+  name: vsphere-machine
+spec:
+  metadata:
+    name: node1
+    labels:
+      foo: "bar"
+  providerConfig:
+    sshPublicKeys:
+      - "<< YOUR_PUBLIC_KEY >>"
+    cloudProvider: "vsphere"
+    cloudProviderSpec:
+      templateVMName: ubuntu-template
+      username: '<VSPHERE_USERNAME>'
```

Comment:
please change it to `<< VSPHERE_USERNAME >>` as this seems to be our "naming convention"
this also applies to `<< VSPHERE_ADDRESS >>`